lambda/update/parse/changes.go

```diff
@@ -154,7 +154,7 @@ func oldEqualsExisting(old any, existing any) bool {
 			return false
 		}
 
-		return oldTime.Equal(*v)
+		return oldTime.Equal(v.Truncate(time.Second))
```

Comment:
the time.RFC3339 above should be time.RFC3339Nano, then you shouldn't need this

Reply:
Gah, time! I'm sure I'd tried that before but works out fine now